tcmur_cmd_handler.c

```diff
@@ -553,6 +553,8 @@ int tcmur_cmd_handler(struct tcmu_device *dev, struct tcmulib_cmd *cmd)
 		/* Try to passthrough the default cmds */
 		if (rhandler->handle_cmd)
 			ret = handle_passthrough(dev, cmd);
+		else
+			tcmu_warn("unknown command %x\n", cdb[0]);
```

Comment:
For the tcmur_cmd_passthrough_handler, this will only for the passthrough type. Yes, there should add the same warning message too.

But for the none passthrough type the warning message is need too.

So I will update this patch with adding warning message in the passthough handler.